app/controllers/admin/users_controller.rb

```diff
@@ -1,10 +1,26 @@
 class Admin::UsersController < AdminController
   def index
-    @users = if params[:query].present?
-               search.perform
-             else
-               user_index_scope.order('created_at desc').paginate(page: params[:page])
-             end
+    (@filterrific = initialize_filterrific(
+      User,
+      params[:filterrific],
+      select_options: {
+        sorted_by: User.options_for_sorted_by,
+        with_volunteer_type: User.volunteer_types.keys.map { |volunteer_type| [volunteer_type.humanize, volunteer_type] }
```

Comment:
Would it help with readability to move `User.volunteer_types.keys.map { |volunteer_type| [volunteer_type.humanize, volunteer_type]` into a private method? Maybe called something like `volunteer_select_types` or something?

Reply:
Ok yes, I might need to pair with you on this part to make sure I understand what you mean.